### PR TITLE
Create expander.yml

### PR DESCRIPTION
**Pull request by jnelson1**
First draft for expander codes entry for CMSC858G assignment.

### CODE REVIEW

File: codes/bits/expander.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+name: 'Expander code'
+
+# First reference with the code (optional).
+introduced: '\cite{doi:10.1109/18.556667}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'Expander codes are binary linear codes whose parity check matrices are derived from the adjacency matrix of bipartite expander graphs. In particular, the rows of the parity check matrix correspond to the right nodes of the bipartite graph and the columns correspond to the left nodes. The codespace is equivalent to all subsets of the left nodes in the graph which have an even number of edges going into every right node of the graph. Since the expander graph is only left regular, these codes do not qualify as LDPC codes. Expander codes are important because they have the desirable property that their rate and relative distance are constant. In addition, they admit efficient encoding and decoding algorithms.'
+
+protection: 'Bit flip errors of weight at most (d-1)/2 where d is the distance of the code and is linear in n, the number of physical bits.'
```
Comment (valbert4) [inline codes/bits/expander.yml:35]: \(d\) and \(n\) math symbols

File: codes/bits/expander.yml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+  # The fields below relate to specific properties of the code that are worth
+  # noting. In some cases (see below), there might be multiple pieces of
+  # information associated with a field (e.g., mutliple encoders/decoders,
+  # multiple facts about fault tolerance, etc.); in such cases, the field
+  # accepts a list of text entries, with one item per piece of information.
+  #
+  # Any fields below should be omitted if they do not apply.
+
+  rate: 'A (n, m, D, γ, α) bipartite expander graph is defined as a D-left-regular graph with n left nodes, and m right nodes such that for any subset of left nodes S of size at most γn the neighborhood N(S) is at least α|S|. Given a (n, m, D, γ, (1-ε)D) expander graph, the corresponding expander code has rate of 1 - m/n and a distance of at least 2(1-ε)γn for any ε < 1/2. Explicit constructions for expander graphs with any ratio n/m are known where D = polylog(n/m), γ = Ω(1/D) and arbitrary ε \cite{doi:10.1145/509907.510003}.'
```
Comment (valbert4) [inline codes/bits/expander.yml:49]: Nice

**Comment by jnelson1** [comment]
I addressed your comments in the last commit. Let me know if you have any trouble viewing this.

File: codes/bits/expander.yml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+#   - 'Links to code tables, github, GAP algebra packages, more papers \cite{arXiv:####.#####}.'
+#   - 'Connections to physics; based on \(H^4(\mathcal{BG},\mathbb{Z})\)'
+#   - 'Can reference other codes, but try to put such connections in the relations fields.'
+#   - 'Anything noteworthy that''s not above or below.'
+
+relations:
+  parents:
+    - code_id: binary_linear
+      #detail: 'code_id1 is smallest code family that includes this code that maintains the same physical space structure'
+    # - code_id: tanner
```
Comment (valbert4) [inline codes/bits/expander.yml:102]: I've added a blurb; you can uncomment.

**Pull request closed without merging**